### PR TITLE
remove steps that generates embedding plot in step

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-862

#### Link to staging deployment URL 

In UI repo : https://github.com/biomage-ltd/ui/pull/230

#### Links to any Pull Requests related to this

UI - https://github.com/biomage-ltd/ui/pull/230
API - https://github.com/biomage-ltd/api/pull/99
Pipeline - https://github.com/biomage-ltd/pipeline/pull/53

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

- Remove steps that carries out embedding and clustering in pipeline

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Review by ivababukova — COMMENTED** [review]
lgtm in terms of code, but I will wait for testing it in staging, making sure it all works before approving

File: qc-runner/src/configureEmbedding.r [hunk 1]
```diff
@@ -42,62 +42,12 @@ task <- function(scdata, config, task_name, sample_id){
     if (is.null(config$enabled) || as.logical(toupper(config$enabled))){
```
Comment (ogibson-biomage) [inline qc-runner/src/configureEmbedding.r:42]: This isn't needed either

**Review by ogibson-biomage — CHANGES_REQUESTED** [review]
We can delete all the code from this pipeline step. Even if we don't want to remove the step itself, it can just be an empty function

**Comment by aerlaut** [comment]
Will be merged in with other changes in the pipeline.

**Pull request closed without merging**